vhdbuilder/packer/test/vhd-content-test.ps1

```diff
@@ -183,6 +183,7 @@ function Test-ImagesPulled {
                 "mcr.microsoft.com/oss/kubernetes-csi/secrets-store/driver:v0.0.19",
                 "mcr.microsoft.com/oss/azure/secrets-store/provider-azure:0.0.12",
                 "mcr.microsoft.com/k8s/csi/azuredisk-csi:v1.0.0",
+                "mcr.microsoft.com/k8s/csi/azuredisk-csi:v1.1.0",
                 "mcr.microsoft.com/k8s/csi/azurefile-csi:v1.0.0")
```

Comment:
why do we have this duplicate

Reply:
there is a problem in integrating azuredisk 1.0.0 with aks